core-bundle/src/Resources/contao/library/Contao/StringUtil.php

```diff
@@ -584,7 +584,7 @@ function (array $matches) use ($arrData)
 
 		$evaluateExpression = function ($strExpression) use ($arrData)
 		{
-			if (!preg_match('/^([^=!<>\s]+)([=!<>]+)(.+)$/is', $strExpression, $arrMatches))
+			if (!preg_match('/^([^=!<>\s]+)[\s]*([=!<>]+)[\s]*(.+)$/is', $strExpression, $arrMatches))
```

Comment:
I would change this to:

```suggestion
			if (!preg_match('/^([^=!<>\s]+)[ \t]*([=!<>]+)(.+)$/is', $strExpression, $arrMatches))
```

and remove the whitepace around the value in line 594:

```php
$strValue = trim($arrMatches[3], " \t");
```

Reply:
I would only allow spaces and not tabs.

Reply:
I have added the proposed changes. @ausi @leofeyer can you have a look at it again?

I also tested it via the registration module.